lms/djangoapps/verify_student/views.py

```diff
@@ -1376,12 +1376,23 @@ def get(self, request):
         """
         status, _ = SoftwareSecurePhotoVerification.user_status(request.user)
 
+        expiration_datetime = SoftwareSecurePhotoVerification.get_expiration_datetime(request.user)
+        can_reverify = False
+        if expiration_datetime:
```

Comment:
@schenedx I can't see any other place with the same `if`. Can you please explain your requirement.